docs/UI/SetGlobalProperties/index.md

```diff
@@ -76,33 +62,53 @@ _**Notes for HMI expected behavior:**_
 
 ### Response
 
-|Result |Description |Message type WebSocket|Message type D-Bus|Message Params|
-|:------|:-----------|:---------------------|:-----------------|:-------------|
-|Success|SUCCESS: HMI has set the requested properties.|JSON response|Method return|code: 0|
-|Failure|INVALID_ID:appID is not valid (e.g.does not exist)|JSON response|Method return|code: 13|
-|Failure|INVALID_DATA: The data sent is invalid (invalid JSON syntax or parameters out of bounds or of wrong type)|JSON response|Method return|code: 11|
-|Failure|GENERIC_ERROR: The unknown issue occurred or other codes are not applicable.|JSON response|Method return|code: 22|
+!!! MUST
 
-!!! NOTE
-In case HMI does not respond SDL's request during SDL-default timeout (10 sec), SDL will return GENERIC_ERROR  result code to the corresponding mobile app's request. Please see [Result Enumeration] for all SDL-supported codes.
+1. Store the information and associate it with `appID`.
+
+_Note: Initially, the appID together with other application-related information is provided by SDL within UpdateAppList or OnAppRegistered RPCs._
+
```

Comment:
```suggestion
1. Store the information and associate it with `appID`.
    * _Note:_ Initially, the appID together with other application-related information is provided by SDL within UpdateAppList or OnAppRegistered RPCs.
```
This might format better. As it is now, it breaks up the numbered list into several smaller numbered lists in the dev portal